spi-ch341.c

```diff
@@ -59,7 +59,7 @@ struct spi_gpio {
 };
 
 struct ch341_spi {
-	struct spi_master *master;
+	struct spi_controller *master;
```

Comment:
This will break the build on older kernels that do not have the new naming. 
Add an ifdef for older kernels when spi_controller is defined.